rasa/cli/x.py

```diff
@@ -112,10 +112,21 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs.get("wait_time_between_pulls")
+    custom_url = endpoints.model.url
+
+    if custom_url and custom_url != rasa_x_url:
```

Comment:
You have to declare a variable (e.g. `default_rasax_model_server_url` with `{rasa_x_url}/projects/default/models/tag/production instead`) and then compare to this and then you can also reuse the `default_rasax_model_server_url` in your prints

Reply:
check this now I think this is how you want it now we have changed it a few times now so trying to keep my head straight on what we are doing there.